### DISCUSSION TITLE
Assembly listing-How do I get the compiler to produce it?

### DISCUSSION THREAD

**Discussion by djsbriscoe** [chipKIT32/chipKIT-core#486] [Q&A]
Now I've got a few sketches compiled on the Arduino IDE V2, I'm curious about what kind of assembly code is produced by the compiler for my uC32 (pic32mx340f512h based) and Uno32 (pic32mx320f128h based). Can I edit some kind of file to append a switch to a command line somewhere? Or would I have to use the arduino-cli option using a command shell (I’m using Windows 10 Pro 64 bit). Where can I find out how to use Arduino CLI? Thanks.

**Comment by JacobChrist** [comment]
I think you can hold down control or shit or some combination of control-shift-alt to (when compiling) keep the IDE from wiping out the temp file but its been a while since I've done this and I don't know if its on V2.

**Comment by majenkotech** [comment]
You could always take the compiled .elf file and use `pic32-objdump` to create an assembly listing (also interspersed with the C source code too).
